src/domFacade/DomFacade.ts

```diff
@@ -1,135 +1,485 @@
+import {
+	AttributeNodePointer,
+	TinyAttributeNode,
+	CharacterDataNodePointer,
+	TinyCharacterDataNode,
+	ChildNodePointer,
+	ElementNodePointer,
+	TinyElementNode,
+	NodePointer,
+	ParentNodePointer,
+	TinyParentNode,
+	ProcessingInstructionNodePointer,
+	TinyChildNode,
+	GraftPoint,
+	TinyNode
+} from '../domClone/Pointer';
+import { Attr, CharacterData, Element, Node } from '../types/Types';
 import {
 	ConcreteAttributeNode,
-	ConcreteCharacterDataNode,
 	ConcreteChildNode,
-	ConcreteElementNode,
-	ConcreteNode,
 	ConcreteParentNode,
 	NODE_TYPES
 } from './ConcreteNode';
 import IDomFacade from './IDomFacade';
-import IWrappingDomFacade from './IWrappingDomFacade';
+
+const isTinyNode = node => node.isTinyNode;
```

Comment:
Why do we need this as a separate function? I don't see the need to change a one liner to a one liner.